packages/cardano-services-client/src/UtxoProvider/BlockfrostUtxoProvider.ts

```diff
@@ -1,20 +1,18 @@
-import { BlockfrostProvider } from '../../util/BlockfrostProvider/BlockfrostProvider';
-import {
-  BlockfrostToCore,
-  blockfrostToProviderError,
-  fetchByAddressSequentially
-} from '@cardano-sdk/cardano-services-client';
+import { BlockfrostProvider, BlockfrostToCore, fetchByAddressSequentially } from '../blockfrost';
 import { Cardano, Serialization, UtxoByAddressesArgs, UtxoProvider } from '@cardano-sdk/core';
 import { PaginationOptions } from '@blockfrost/blockfrost-js/lib/types';
 import { Responses } from '@blockfrost/blockfrost-js';
-import { Schemas } from '@blockfrost/blockfrost-js/lib/types/open-api';
 
 export class BlockfrostUtxoProvider extends BlockfrostProvider implements UtxoProvider {
-  protected async fetchUtxos(addr: Cardano.PaymentAddress, pagination: PaginationOptions): Promise<Cardano.Utxo[]> {
-    const utxos: Responses['address_utxo_content'] = (await this.blockfrost.addressesUtxos(
-      addr.toString(),
-      pagination
-    )) as Responses['address_utxo_content'];
+  protected async fetchUtxos(
+    addr: Cardano.PaymentAddress,
+    { page, count, order }: PaginationOptions
+  ): Promise<Cardano.Utxo[]> {
+    let queryString = `addresses/${addr.toString()}/utxos`;
+    if (page) queryString += `?page=${page}`;
+    if (count) queryString += `?count=${count}`;
+    if (order) queryString += `?order=${order}`;
```

Comment:
Would be nice to extract a util that is shared with [fetchSequentially](https://github.com/input-output-hk/cardano-js-sdk/blob/f9d460a3fd97481bfb2fa5719a838b3521283907/packages/cardano-services-client/src/blockfrost/util.ts#L23) and [other providers that do this](https://github.com/input-output-hk/cardano-js-sdk/pull/1532/files#diff-ac7949b8cb3a04190a1c033740ff86720a98f0af72c6513fbd71d34d146b540dR508-R510)

Reply:
Done in e24be482d3380ad9d0c2b590e5296aac7cec39f1